frontend/apps/web/src/index.css

```diff
@@ -0,0 +1,65 @@
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+  color: #213547;
+  background-color: #ffffff;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  place-items: center;
+  min-width: 320px;
+  min-height: 100vh;
+}
+
+#root {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
+}
+
+.dashboard {
+  padding: 2rem;
+  border-radius: 8px;
+  border: 1px solid #e2e8f0;
+  background: #f8fafc;
+}
+
+.dashboard h1 {
+  margin: 0 0 1rem 0;
+  color: #1e293b;
+}
+
+.loading {
+  color: #64748b;
+  font-style: italic;
+}
+
+.error {
+  color: #dc2626;
+  background: #fee2e2;
+  padding: 1rem;
+  border-radius: 4px;
+  border: 1px solid #fca5a5;
+}
+
+.data-display {
+  background: white;
+  padding: 1rem;
+  border-radius: 4px;
+  border: 1px solid #d1d5db;
+  margin: 1rem 0;
+}
+
+.data-display h3 {
+  margin: 0 0 0.5rem 0;
+  color: #374151;
+}
+
+.data-display p {
+  margin: 0.25rem 0;
+  color: #6b7280;
+}
```

Comment:
validate best practices and define is its the best place to keep this file or can be a better one?